release.yaml

```diff
@@ -250,4 +250,3 @@ api:
     bom:          false
     javadoc:      false
     isolated:     true
-    codecoverage: true
```

Comment:
Sorry to be nit picky but it looks like the script has cut off the last line here

Reply:
Thanks, should be sorted now!